sharing_portal/views.py

```diff
@@ -205,6 +205,32 @@ def _fetch_artifacts(filters):
     )
 
 
+def _delete_artifact_version(request, version):
+    if not version.doi:
+        try:
+            version.delete()
+        except Exception as e:
+            messages.add_message(
+                request,
+                messages.ERROR,
+                f"Error deleting artifact version {version}: {str(e)}",
```

Comment:
In general we should not be showing exception text to end-users. I think it is best to log the exception internally and show them a generic error.